Cargo.toml

```
@@ -10,24 +10,24 @@ edition = "2021"
 repository = "https://github.com/stackabletech/opa-operator"
 
 [workspace.dependencies]
-axum = "0.6"
-built = { version = "0.6", features = ["chrono", "git2"] }
+axum = "0.7"
+built = { version ="0.7", features = ["chrono", "git2"] }
 clap = "4.3"
 derivative = "2.2"
 fnv = "1.0"
 futures = { version = "0.3", features = ["compat"] }
-hyper = "0.14"
+hyper = "1.3"
 indoc = "2.0"
 moka = { version = "0.12", features = ["future"] }
 pin-project = "1.1"
 product-config = { git = "https://github.com/stackabletech/product-config.git", tag = "0.6.0" }
-reqwest = { version = "0.11", features = ["json"] }
+reqwest = { version ="0.12", features = ["json"] }
 semver = "1.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-snafu = "0.7"
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.64.0" }
-strum = { version = "0.25", features = ["derive"] }
+snafu = "0.8"
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "stackable-operator-0.67.0" }
```

Sorry, there is already a new version available
```suggestion
stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "stackable-operator-0.67.1" }
```